src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+            } else if(line.startsWith("event")) {
+                createEventTask(line , taskLetter ,list);
+            } else {
+                System.out.println("    Ps didn't understand command , please refer to the command guide");
+                printCommandGuide();
+            }
+            in = new Scanner(System.in);
+            line = in.nextLine();
+        }
+        printBye();
+    }
+
+    public static void printGreeting() {
+        String logo = "      ____        _        \n"
+                + "     |  _ \\ _   _| | _____ \n"
+                + "     | | | | | | | |/ / _ \\\n"
+                + "     | |_| | |_| |   <  __/\n"
+                + "     |____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("    _____________________________________________________________");
+        System.out.println("    Hello! I'm Duke");
+        printCommandGuide();
+    }
+
+
+    public static void printCommandGuide(){
+        //prints list of commands
+        System.out.println("    Command Guide");
+        System.out.println("    use \"todo\" (with a space) to add a task without any date or time attached to it");
+        System.out.println("    use \"deadline\" (with a space) and \"/by\" to add a task to be done before a certain date");
+        System.out.println("    use \"event\" (with a space) and \"/at\" to add a task that start and ends at a specific time");
+        System.out.println("    use \"list\" to show task list");
+        System.out.println("    use \"done\" (with a space) followed by task number to mark a task as done");
+        System.out.println("    use \"bye\" to exit the chat bot");
+        System.out.println("    What can I do for you?");
+        System.out.println("    _____________________________________________________________");
+    }
+
+    /**
+     *Creates a new ToDo object of Task class
+     *adds ToDo Object to list of tasks
+     *prints a message to tell the user ToDo object has been added to TaskList
+     *
+     * @param line the input entered by user
+     * @param taskLetter the letter denoting the type of object
+     * @param list list of tasks
+     * */
+    public static void createToDoTask(String line , char taskLetter , TaskList list){
+        Task task = new ToDo(line.substring(5));
+        taskLetter = Character.toUpperCase(line.charAt(0));
+        list.addTaskList(task);
+        printEcho(task ,list);
+    }
+
+    /**
+     *Creates a new Deadline object of Task class
+     *adds Deadline Object to list of tasks
+     *prints a message to tell the user Deadline object has been added to TaskList
+     *
+     * @param line the input entered by user
+     * @param taskLetter the letter denoting the type of object
+     * @param list list of tasks
+     * */
+    public static void createDeadlineTask(String line , char taskLetter , TaskList list){
+        String[] deadline = line.split("/by");
```

Comment:
maybe change variable name to `deadlines`? since it is an array so the name should be plural